### PR TITLE
regenerated languages

### PR DESCRIPTION
**Pull request by cdietrich**
regenerated languages
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/antlr/services/Bug301935TestLanguageGrammarAccess.java [hunk 1]
```diff
@@ -38,7 +38,8 @@
 		//	value2=ID WS;
 		@Override public ParserRule getRule() { return rule; }
 		
-		//name=ID WS value=ID NL value2=ID WS
+		//name=ID WS value=ID NL
+		//value2=ID WS
```
Comment (szarnekow) [inline org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/antlr/services/Bug301935TestLanguageGrammarAccess.java:42]: To me looks more and more like an off-by-one error in the formatter.

File: org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/terminalrules/services/Bug297105TestLanguageGrammarAccess.java [hunk 2]
```diff
@@ -148,14 +148,14 @@
 		private final RuleCall cEXT_INTTerminalRuleCall_2_0 = (RuleCall)cAlternatives_2.eContents().get(0);
 		private final RuleCall cINTTerminalRuleCall_2_1 = (RuleCall)cAlternatives_2.eContents().get(1);
 		
-		//Real hidden():
-		//	INT? '.' (EXT_INT | INT);
+		//Real hidden ():
+		//	INT ? '.' (EXT_INT | INT);
```
Comment (szarnekow) [inline org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/terminalrules/services/Bug297105TestLanguageGrammarAccess.java:152]: @ArneDeutsch Unexpected space before the cardinality `?`.

**Review by szarnekow — CHANGES_REQUESTED** [review]
No need to repeat the things already said in the other merge requests.

File: org.eclipse.xtext.testlanguages/src-gen/org/eclipse/xtext/testlanguages/xtextgrammar/services/XtextGrammarTestLanguageGrammarAccess.java [hunk 3]
```diff
@@ -360,14 +361,16 @@
 		private final Keyword cSemicolonKeyword_5 = (Keyword)cGroup.eContents().get(5);
 		
 		//ParserRule:
-		//	name=ID ('returns' type=TypeRef)? (definesHiddenTokens?='hidden' '(' (hiddenTokens+=[AbstractRule] (','
-		//	hiddenTokens+=[AbstractRule])*)? ')')? ':'
+		//	name=ID ('returns' type=TypeRef)? (definesHiddenTokens?='hidden' '(' (hiddenTokens+=[AbstractRule] (',' hiddenTokens+=
```
Comment (cdietrich) [inline org.eclipse.xtext.testlanguages/src-gen/org/eclipse/xtext/testlanguages/xtextgrammar/services/XtextGrammarTestLanguageGrammarAccess.java:364]: bodus formatting

**Review by szarnekow — APPROVED** [review]
It is how it is.

File: org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/antlr/services/Bug301935TestLanguageGrammarAccess.java (hunk 1 above)
Comment (ArneDeutsch) [reply org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/antlr/services/Bug301935TestLanguageGrammarAccess.java:42]: But this is expected ... newline is preserved ...

File: org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/terminalrules/services/Bug297105TestLanguageGrammarAccess.java (hunk 2 above)
Comment (ArneDeutsch) [reply org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/terminalrules/services/Bug297105TestLanguageGrammarAccess.java]: Created a unit test, seems to work as expected now.

File: org.eclipse.xtext.testlanguages/src-gen/org/eclipse/xtext/testlanguages/xtextgrammar/services/XtextGrammarTestLanguageGrammarAccess.java (hunk 3 above)
Comment (ArneDeutsch) [reply org.eclipse.xtext.testlanguages/src-gen/org/eclipse/xtext/testlanguages/xtextgrammar/services/XtextGrammarTestLanguageGrammarAccess.java:364]: Added a unit test and it does what I have expected, looks good for me. Maybe it is the line breaks that confuses you? They are already in the origianal rule definition and are preserved.

File: org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/terminalrules/services/Bug297105TestLanguageGrammarAccess.java (hunk 2 above)
Comment (szarnekow) [reply org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/terminalrules/services/Bug297105TestLanguageGrammarAccess.java]: Roger that

File: org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/antlr/services/Bug301935TestLanguageGrammarAccess.java (hunk 1 above)
Comment (szarnekow) [reply org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/antlr/services/Bug301935TestLanguageGrammarAccess.java:42]: Roger that

**Pull request merged**